diff --git a/exceptions/http.exception.d.ts b/exceptions/http.exception.d.ts
index e56ede0b858313355b5c3950fbac2ba7b5b90fc8..0b1144345becd12f4f1bd2034419a5b3691f8e75 100644
--- a/exceptions/http.exception.d.ts
+++ b/exceptions/http.exception.d.ts
@@ -1,7 +1,7 @@
 import { HttpExceptionBody, HttpExceptionBodyMessage } from '../interfaces/http/http-exception-body.interface';
 export interface HttpExceptionOptions {
     /** original cause of the error */
-    cause?: unknown;
+    cause?: Error;
     description?: string;
 }
 export interface DescriptionAndOptions {
@@ -55,7 +55,7 @@ export declare class HttpException extends Error {
      * @param options An object used to add an error cause.
      */
     constructor(response: string | Record<string, any>, status: number, options?: HttpExceptionOptions);
-    cause: unknown;
+    cause: undefined;
     /**
      * Configures error chaining support
      *
