diff --git a/index.d.ts b/index.d.ts
index 0c63c8df26c8dec41937bfb7af33fafc2f59d454..b180242b829710d690de9fede42d7272cbe55f2f 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -1,5 +1,6 @@
 import * as v1alpha2 from './ory/keto/relation_tuples/v1alpha2'
 
 declare module '@ory/keto-grpc-client' {
+  // @ts-ignore
   export default v1alpha2
 }
diff --git a/ory/keto/relation_tuples/v1alpha2/index.d.ts b/ory/keto/relation_tuples/v1alpha2/index.d.ts
index 7e52c454dddc9314957b4015d1b0166f5f243c0d..27946e94c5eaf13dd77a58701b9fba756956afe9 100644
--- a/ory/keto/relation_tuples/v1alpha2/index.d.ts
+++ b/ory/keto/relation_tuples/v1alpha2/index.d.ts
@@ -8,6 +8,7 @@ import * as expandService from './expand_service_grpc_pb'
 import * as read from './read_service_pb'
 import * as readService from './read_service_grpc_pb'
 declare module '@ory/keto-grpc-client/ory/keto/relation_tuples/v1alpha2' {
+  // @ts-ignore
   export {
     relationTuples,
     write,
